src/components/layout/grid/GridComponent.scss

```
@@ -25,7 +25,7 @@
   .club-name {
     letter-spacing: 0.03em;
     font-size: 22px;
-    font-family: Qanelas Soft;
+    /* font-family: Qanelas Soft; */
```

I feel like we should just delete this rather than leave a bunch of commented out code in the codebase, but lmk if anyone disagrees.

```suggestion
    /* font-family: Qanelas Soft; */
```